### PR TITLE
@benisgold/helpers-memoization

### PR DESCRIPTION
**Pull request by benisgold**
This PR applies `memoFn` from https://github.com/rainbow-me/rainbow/pull/2637 to relevant functions within src/utils.

### CODE REVIEW

File: src/helpers/time.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const getMinimalTimeUnitStringForMs = (
   short = true,
   plural
 ) => {
+  console.log("zooooo");
```
Comment (skylarbarrera) [inline src/helpers/time.js:37]: ser i love zoos but not with my rainbo's

File: src/helpers/utilities.ts [hunk 2]
```diff
@@ -145,8 +145,10 @@ export const formatInputDecimals = (
 export const convertHexToString = (hex: BigNumberish): string =>
   new BigNumber(hex).toFixed();
 
-export const convertStringToHex = (stringToConvert: string): string =>
-  new BigNumber(stringToConvert).toString(16);
+export const convertStringToHex = (stringToConvert: string): string => {
+  console.log("zoopy");
```
Comment (skylarbarrera) [inline src/helpers/utilities.ts:149]: soupy

**Review by skylarbarrera — APPROVED** [review]
lgtm after rm logs & linting

**Review by terrysahaidak — CHANGES_REQUESTED** [review]
All is good except for my coment

**Pull request closed without merging**